hack/lib/util.sh

```diff
@@ -700,8 +700,7 @@ function kube::util::ensure_dockerized {
 function kube::util::ensure-gnu-sed {
   # NOTE: the echo below is a workaround to ensure sed is executed before the grep.
   # see: https://github.com/kubernetes/kubernetes/issues/87251
-  sed_help="$(LANG=C sed --help 2>&1)"
-  if echo "${sed_help}" | grep -q "GNU\|BusyBox"; then
+  if echo "$(LANG=C sed --help 2>&1)" | grep -q "GNU\|BusyBox"; then
```

Comment:
this seems like a very useless use of echo?

Reply:
Yep, I removed the ``echo``.
But there have some comments above, I'm not sure without the ``echo`` command is there any known issues?